nym-wallet/src/pages/bonding/node-settings/settings-pages/general-settings/InfoSettings.tsx

```diff
@@ -0,0 +1,246 @@
+import { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Button, Divider, Typography, TextField, Grid, Alert, IconButton, CircularProgress } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CloseIcon from '@mui/icons-material/Close';
+import { isMixnode } from 'src/types';
+import { updateMixnodeConfig } from 'src/requests';
+import { TBondedMixnode, TBondedGateway } from 'src/context/bonding';
+import { SimpleModal } from 'src/components/Modals/SimpleModal';
+import { bondedInfoParametersValidationSchema } from 'src/components/Bonding/forms/mixnodeValidationSchema';
+
+export const InfoSettings = ({ bondedNode }: { bondedNode: TBondedMixnode | TBondedGateway }) => {
+  const [open, setOpen] = useState(true);
+  const [openConfirmationModal, setOpenConfirmationModal] = useState<boolean>(false);
+
+  const theme = useTheme();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting, isDirty, isValid },
+  } = useForm({
+    resolver: yupResolver(bondedInfoParametersValidationSchema),
+    mode: 'onChange',
+    defaultValues: isMixnode(bondedNode) ? bondedNode : {},
+  });
+
+  const onSubmit = async (data: {
+    host?: string;
+    version?: string;
+    mixPort?: number;
+    verlocPort?: number;
+    httpApiPort?: number;
+  }) => {
+    const { host, version, mixPort, verlocPort, httpApiPort } = data;
+    if (host && version && mixPort && verlocPort && httpApiPort) {
+      const MixNodeConfigParams = {
+        host,
+        mix_port: mixPort,
+        verloc_port: verlocPort,
+        http_api_port: httpApiPort,
+        version,
+      };
+      try {
+        await updateMixnodeConfig(MixNodeConfigParams);
+        setOpenConfirmationModal(true);
+      } catch (error) {
+        console.error(error);
```

Comment:
you should use `Console.error` instead (to have a trace of the error in tauri logs as well)